libs/single-spa-angular/internals/package.json

```diff
@@ -1,10 +0,0 @@
-{
-  "ngPackage": {
-    "$schema": "../../node_modules/ng-packagr/ng-package.schema.json",
-    "lib": {
-      "entryFile": "src/index.ts",
-      "umdId": "single-spa-angular-internals",
```

Comment:
This was removed, does this mean that single-spa-angular will no longer produce umd bundles? If so, that may break things for people relying on it loading as an in-browser systemjs module

Reply:
@joeldenning Indeed, Angular 13 makes the format by default the usage of ESM as described here: [Angular Package Format](https://angular.io/guide/angular-package-format). But I am not an expert in bundles/webpack so I don't know what is the impact on single-spa?!

I know there are some repositories for allowing angular to be imported using systemJs (https://github.com/esm-bundle/angular__core) but is this format not compatible with systemJs?

@arturovt @chucknelson I think we should have a look at this question.

Reply:
after a quick, this is indeed bad:
- https://github.com/evanw/esbuild/issues/192
- https://github.com/evanw/esbuild/pull/255
- https://github.com/evanw/esbuild/issues/507

Reply:
@jogelin - I think even if `esbuild` supported outputting in UMD bundles, `ng-packagr` would still need to support outputting libraries in that as well. To my knowledge there are no plans to customize build output formats or anything, though, since `ng-packagr` is all about conforming to whatever the latest Angular Package Format (APF) spec is.

Maybe there are some clever workarounds, post-processing or something to get Angular 13+ built libraries to work as in-browser modules, but I'm not sure - nor do I know if that'll seem "right" in any way, might feel too hacky?

My experience with shared Angular has been spotty already (see https://github.com/single-spa/single-spa.js.org/issues/490), so maybe I'm missing something. @arturovt might know more.

From my experience thus far, tree-shaking in Angular works pretty well for keeping MFE bundle sizes small, so my teams have used "plain" Typescript/Javascript libraries for in-browser loading, which are utilized by native Angular libraries or apps that use them via webpack externals.